### PR TITLE
Do not recreate object when changing frozen state

### PR DESCRIPTION
**Pull request by lopezloo**
Fixes frozen related
crashes described in #[9199](https://bugs.mtasa.com/view.php?id=9199) & #[8765](https://bugs.mtasa.com/view.php?id=8765) and generaly works better - object won't blink and rotation stay the same.

### CODE REVIEW

File: MTA10/mods/shared_logic/CClientObject.cpp [hunk 1]
```diff
@@ -502,16 +510,14 @@ void CClientObject::Create ( void )
                 // Add XRef
                 g_pClientGame->GetGameEntityXRefManager ()->AddEntityXRef ( this, m_pObject );
 
-                // If set to true,this has the effect of forcing the object to be static at all times
-                m_pObject->SetStaticWaitingForCollision ( m_bIsStatic );
```
Comment (qaisjp) [inline MTA10/mods/shared_logic/CClientObject.cpp:506]: What does the removal of this line achieve?

Comment (lopezloo) [inline MTA10/mods/shared_logic/CClientObject.cpp:506]: I replaced this with SetFrozen below. Generally this _StaticWaitingForCollision_ flag has effect only for _true_ so recreating object was required.

If you are worried about this initial freeze until something touch object - it's still here.

**Comment by lopezloo** [comment]
I found issue.
crun obj = Object(1222, 0, 0, 4)
crun obj.frozen = true
crun obj.frozen = false

Should stay frozen until something touch it but goes down because speed got changed.

**Comment by lopezloo** [comment]
Ok. I added check to avoid this plus modified turn speed change func. Sometimes objects was still rotating after froze, now it's okay.

BTW. I created comparision.
Before:
old frozing
After:
new frozing

**Comment by qaisjp** [comment]
I like the new functionality. May have issues with BC though. Don't worry about it for now.

Side question: please can you share the source for the script shown in those gifs?

**Comment by lopezloo** [comment]
http://pastebin.com/uUCaHeJt
/bo - boom
/fo - (un)freeze

**Pull request closed without merging**